pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -283,7 +283,22 @@ self: super: {
   persistent-zookeeper = dontCheck super.persistent-zookeeper;
   pocket-dns = dontCheck super.pocket-dns;
   postgresql-simple = dontCheck super.postgresql-simple;
-  postgrest = dontCheck super.postgrest;
+  
+  postgrest = let 
+      postgrest_pre = addBuildDepends super.postgrest [super.configurator super.heredoc super.insert-ordered-containers super.lens super.lens-aeson super.protolude self.swagger2];
+      in
+  dontHaddock (dontCheck (( overrideCabal postgrest_pre (drv: {
+    enableSharedExecutables=false;
+    version = "0.3.2-head";
```

Comment:
Would it be better to add `--allow-newer` to the cabal flags?

`postgrest = dontCheck (appendConfigureFlag super.postgrest "--allow-newer");`